src/sinol_make/util.py

```diff
@@ -204,6 +210,12 @@ def check_version():
     Function that asynchronously checks for new version of sinol-make.
     Writes the newest version to data/version file.
     """
+    python_version = sys.version_info
+    if python_version.minor <= 8:
+        import importlib_resources as importlib
+    else:
+        import importlib.resources as importlib
+
```

Comment:
code duplication?

Reply:
I added a function for importing the library